src/style.css

```diff
@@ -1,3 +1,68 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
```

Comment:
Avoid using tag names for styling except for `html` and `body`. Instead, use class selectors for styling.